src/kvstore/KVStore.h

```diff
@@ -83,6 +84,13 @@ class KVStore {
                                 const std::vector<std::string>& keys,
                                 std::vector<std::string>* values) = 0;
 
+    // Read multiple keys, if key[i] does not exist, the i-th value in return value
+    // would be Status::KeyNotFound
+    virtual ErrorOr<ResultCode, std::vector<Status>> tryGet(GraphSpaceID spaceId,
```

Comment:
Why not reuse multiGet directly?

Reply:
Because we need to return the Status, so we can judge whether the key exists. If we reuse `multiGet`, the value of key_not_exist would be empty, it is hard to tell whether the value is really empty or the key doesn't exist.

Reply:
I mean modify the multiGet interface directly...

Reply:
ok